Extensions/fullScreen.js

```diff
@@ -121,6 +121,121 @@ const translations = {
         tvBtnDesc: "TV Mode Display",
         fullscreenBtnDesc: "Full Screen",
     },
+    const translations = {
+    "de--DE": {
+        langName: "Deutsch",
+        context: {
+            queue: "Wiedergabe aus der Warteschlange",
+            track: "Titel spielen",
+            album: "Abspielen von Album",
+            artist: "Abspielen von Künstler",
+            playlist: "Abspielen von Playlist",
+            playlistFolder: "Wiedergabe aus dem Playlist-Ordner",
+            search: "Wiedergabe aus Suche",
+            searchDest: "Lieder",
+            collection: "Abspielen aus Sammlung",
+            likedSongs: "Lieblingssongs",
+            trackRadio: "Abspielen von Titel Radio",
+            artistRadio: "Abspielen von Künstler Radio",
+            albumRadio: "Abspielen von Alben Radio",
+            playlistRadio: "Abspielen von Playlist Radio",
+        },
+        upnext: "ALS NÄCHSTES",
+        unknownArtist: "Künstler (nicht verfügbar)",
+        settings: {
+            switchToTV: "Wechseln zu TV-Modus",
+            switchToFullscreen: "Wechseln zu Standardmodus",
+            tvModeConfig: "Konfiguration des TV-Modus",
+            fullscreenConfig: "Vollbildmodus konfiguration",
+            exit: "Exit",
+
+            pluginSettings: "Plugin Einstellungen",
+            fsHideOriginal: "Stock-Schaltfläche ausblenden (Spotify Premium)",
+            language: "Sprache",
+
+            lyricsHeader: "Liedtext-Einstellungen",
+            lyrics: "Liedtext",
+            lyricsAlignment: {
+                setting: "Liedtext Ausrichtung",
+                left: "Links",
+                center: "Center",
+                right: "Rechts",
+            },
+            lyricsAnimationTempo: "Liedtext Animationtempo",
+
+            generalHeader: "Allgemeine Einstellungen",
+            progressBar: "Fortschrittsanzeige",
+            playerControls: "Wiedegergabe steuerung",
+            trimTitle: "Trimmen Titel",
+            trimTitleUpNext: "Trim Title(Up Next)",
+            showAlbum: {
+                setting: "Album anzeigen",
+                never: "Niemals",
+                always: "Jemand",
+                date: "Veröffentlichungsdatum anzeigen",
+            },
+            showAllArtists: "Alle Künstler anzeigen",
+            icons: "Symbole",
+            songChangeAnimation: "Songwechsel-Animation",
+            fullscreen: "Vollbildmodus",
+
+            extraHeader: "Zusätzliche Funktionalität",
+            backgroundChoice: {
+                setting: "Hintergrundauswahl",
+                color: "Einfarbig",
+                artwork: "Titelbild",
+            },
+            extraControls: "Zusätzliche Steuerelemente",
+            upnextDisplay: "Nächste-titel Anzeige",
```

Comment:
"Nächster Titel-Anzeige"
feel free to improve, my translation here is rather bad